CONTRIBUTING.md

```diff
@@ -84,28 +84,20 @@ The following checklist should be followed when creating a release:
                    `dist/goreleaser/summon-darwin-arm64`
         - Windows: `dist/goreleaser/summon_windows_amd64`
 
-- [ ] Create a draft release:
-  - [ ] Create an [annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_annotated_tags)
-        of the merged commit with the name `v<version>` and push it to the repository.
-        For example: `git tag -a v<version> -m "Release of v<version>"  `
-  - [ ] Using the GitHub UI, create a release using the tag created earlier as the starting point.
-  - [ ] Name the release the same as the tag.
-  - [ ] Include in the release notes all changes from CHANGELOG that are being released.
-    - [ ] Attach the relevant assets to the release, generated previously by `./build`:
-      - [ ] Attach `dist/goreleaser/summon-darwin-a*64.tar.gz` to release.
-      - [ ] Attach `dist/goreleaser/summon-linux-amd64.tar.gz` to release.
-      - [ ] Attach `dist/goreleaser/summon-windows-amd64.tar.gz` to release.
-      - [ ] Attach `dist/goreleaser/summon_v*.rpm` to release.
-      - [ ] Attach `dist/goreleaser/summon_v*.deb` to release.
-      - [ ] Attach `dist/goreleaser/CHANGELOG.md` to release.
-      - [ ] Attach `dist/goreleaser/SHA256SUMS.txt` to the release.
-  - [ ] Publish the release as a "pre-released".
-  
+- [ ] Create an [annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_annotated_tags)
+      of the merged commit with the name `v<version>` and push it to the
+      repository. For example: `git tag -a v<version> -m "Release of v<version>"`
+      This will automatically create a draft release with artifacts attached.
+
+- [ ] Publish the draft as "pre-released".
+
 - [ ] Update homebrew tools
-  - [ ] In [`cyberark/homebrew-tools`](https://github.com/cyberark/homebrew-tools) repo, update
-        the [`summon.rb` formula](https://github.com/cyberark/homebrew-tools/blob/main/summon.rb#L4-L6) with a PR
-        using the file `dist/goreleaser/summon.rb`.
-        
+  - [ ] In [`cyberark/homebrew-tools`](https://github.com/cyberark/homebrew-tools)
+        repo, create a PR to update the [`summon.rb` formula](https://github.com/cyberark/homebrew-tools/blob/main/summon.rb#L4-L6)
```

Comment:
Did you mean to highlight only lines 4-6, and not the entire file?

Reply:
Not sure if that was intended, those lines were already highlighted in the previous version. I'll just leave it for now.